app/src/main/java/de/westnordost/streetcomplete/quests/backup_generator/AddBackupGenerator.kt

```diff
@@ -0,0 +1,58 @@
+package de.westnordost.streetcomplete.quests.backup_generator
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.quests.YesNoQuestForm
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+
+class AddBackupGenerator : OsmFilterQuestType<Boolean>() {
+    val amenities = listOf(
```

Comment:
telephone seems surprising to me here

Reply:
I copied filters from another quest and must have missed that one while deleting not related ones :sweat_smile: It's removed